### ISSUE TITLE
Keyboard accessibility

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#9994]
There are some few things missing at the moment for proper keyboard-only accessibility:

- highlight + tooltip for contacts icon in header
- highlight + tooltip for settings icon in header
- possibly same for notifications
- Upload function in Files add menu not focusable

What else @nextcloud/accessibility @nextcloud/designers?

**Comment by skjnldsv** [comment]
See @jancborchardt! Nextcloud bot can be relevant! ;)

**Comment by kevgathuku** [comment]
Some additional things I can add are:
- It should be easy to visually determine the element with keyboard focus
 - All elements that can be interacted with via a mouse click can also be interacted with via pressing `Enter` / `Spacebar`
Example: 
![output](https://user-images.githubusercontent.com/3721994/41911853-7ec22c2e-7956-11e8-9b88-d437e4e0af23.png)

**Comment by jancborchardt** [comment]
@kevgathuku good points! I incorporated them into the draft of the Accessibility subsite for the Design documentation :) https://github.com/nextcloud/documentation/issues/788
